### PR TITLE
chore!: Normalize repository, dropping node <10.13 support

### PR DESCRIPTION
**Pull request by sttk**
This PR updates this project:

- Updates dependency versions
- Update GitHub Actions files
- Adds, changes and deletes some project files

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 Blaine Bublitz <blaine.bublitz@gmail.com>, Eric Schoffstall <yo@contra.io> and other contributors
+Copyright (c) 2017, 2021 Blaine Bublitz <blaine.bublitz@gmail.com>, Eric Schoffstall <yo@contra.io>
```
Comment (phated) [inline LICENSE]:
```suggestion
Copyright (c) 2017, 2021 Blaine Bublitz <blaine.bublitz@gmail.com> and Eric Schoffstall <yo@contra.io>
```

File: package.json [hunk 2]
```diff
@@ -9,38 +9,43 @@
...
-    "test": "mocha --async-only",
-    "cover": "istanbul cover _mocha",
-    "coveralls": "npm run cover && istanbul-coveralls"
+    "test": "nyc mocha --async-only"
   },
   "dependencies": {
-    "safe-buffer": "^5.1.0",
     "remove-bom-buffer": "^3.0.0",
-    "through2": "^2.0.3"
+    "safe-buffer": "^5.2.1",
```
Comment (phated) [inline package.json]: We'll want to drop safe-buffer completely, it was a holdover for the old buffer support

**Review by phated — CHANGES_REQUESTED** [review]
Sorry, I should have done this as a review. We want to remove safe-buffer in this change.

**Comment by sttk** [comment]
@phated I've removed `safe-buffer`. Please review this PR again.

**Review by phated — APPROVED** [review]
Thanks @sttk! I also opened #4 and #5 for follow ups before we release this next version.

**Pull request merged**